### PR TITLE
CRYO: Support Return to Launcher

### PR DESCRIPTION
**Pull request by ccawley2011**

### CODE REVIEW

File: engines/cryo/cryo.h [hunk 1]
```diff
@@ -82,6 +83,10 @@ class CryoEngine : public Engine {
 	void getMousePosition(int16 *x, int16 *y);
 	void setMousePosition(int16 x, int16 y);
 	bool isMouseButtonDown();
+
+private:
+	int _mouseButton;
+	// byte _keyState[256];
```
Comment (bluegr) [inline engines/cryo/cryo.h:89]: Please remove

File: engines/cryo/eden.cpp [hunk 2]
```diff
@@ -4121,6 +4128,9 @@ void EdenGame::run() {
 			_musicPlayingFlag = false;
 			_musicEnabledFlag = false;
 		}
+
+		if (_vm->getPlatform() == Common::kPlatformMacintosh)
```
Comment (bluegr) [inline engines/cryo/eden.cpp:4132]: Why is this limited to Mac platforms?

**Comment by bluegr** [comment]
Nice changes overall!

File: engines/cryo/eden.cpp (hunk 2 above)
Comment (ccawley2011) [reply engines/cryo/eden.cpp:4132]: The relevant structures are only allocated in `initCubeMac()`, which is only called for the Mac version.

File: engines/cryo/cryo.h (hunk 1 above)
Comment (ccawley2011) [reply engines/cryo/cryo.h:89]: Should I also remove the reference to `_keyState` in `CryoEngine::pollEvents()` as well?

Comment (bluegr) [reply engines/cryo/cryo.h:89]: Yes please. The keyup events should be handled by the event manager... it seems that the _keyState array is a leftover

Comment (ccawley2011) [reply engines/cryo/cryo.h]: OK, done.

**Comment by bluegr** [comment]
Thanks! Merging

**Pull request merged**